docs/BreakingChanges/064 Managed browser hosting controls from the .NET Framework 1.1 and 2.0 are blocked.md

```diff
@@ -0,0 +1,22 @@
+## 64: Managed browser hosting controls from the .NET Framework 1.1 and 2.0
```

Comment:
{
  "Id": "64",
  "Title": "Managed browser hosting controls from the .NET Framework 1.1 and 2.0",
  "Details": "Hosting these controls is blocked in Internet Explorer.",
  "Suggestion": "Internet Explorer will fail to launch an application that uses managed browser hosting controls. The previous behavior can be restored by setting the EnableIEHosting value of the registry subkey `HKLM/SOFTWARE/MICROSOFT/.NETFramework` to `1` for x86 systems and for 32-bit processes on x64 systems, and by setting the `EnableIEHosting` value of the registry subkey `HKLM/SOFTWARE/Wow6432Node/Microsoft/.NETFramework` to `1` for 64-bit processes on x64 systems.",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#web",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 2
}

**Parsed APIs:**
- [ ] Not detectable via API analysis